scripts/lib.ts

```diff
@@ -16,21 +16,12 @@ import {
   walletActions,
 } from "viem";
 import { foundry } from "viem/chains";
+import { BREAD_ADDRESS, BUTTER_ADDRESS } from "../src/constants";
 
-export const anvilConfig = getConfig(31337);
-
-export const anvilAccounts: Array<Hex> = [
-  // mock wallet 2
-  // "0x70997970c51812dc3a010c7d01b50e0d17dc79c8",
-  "0x3c44cdddb6a900fa2b585dd299e03d12fa4293bc",
-  "0x90f79bf6eb2c4f870365e785982e1f101e93b906",
-  "0x15d34aaf54267db7d7c367839aaf71a00a2c6a65",
-  "0x9965507d1a55bcc2695c58ba16fb37d819b0a4dc",
-  "0x976ea74026e726554db657fa54763abd0c3a0aa9",
-  "0x14dc79964da2c08b23698b3d3cc7ca32193d9955",
-  "0x23618e81e3f5cdf7f54c3d65f7fbc0abf5b21e8f",
-  "0xa0ee7a142d267c1f36714e4a8f75612f20a79720",
-];
+// Wrapped into a function so we do not initialize on import, before all contracts are deployed
+export function getAnvilConfig() {
```

Comment:
I was running into a problem: importing `chainConfig.ts` file required certain contracts are already deployed. Since when we run `preDeploy` those contracts do not exist I created "getters" for config so that importing `chainConfig.ts` does not blow things up.

Reply:
I see what was happening now :slightly_smiling_face: